### PR TITLE
Update documentation reference

### PR DESCRIPTION
**Pull request by BrianMMcClain**
## Description

Updated several documentation references to bring them in line with current doc standardization

### CODE REVIEW

**Review by nfagerlund — CHANGES_REQUESTED** [review]
One typo, but otherwise 👍🏼

**Review by nfagerlund — APPROVED** [review]

File: website/docs/d/agent_pool.html.markdown [hunk 1]
```diff
@@ -9,9 +9,7 @@ description: |-
 
 Use this data source to get information about an agent pool.
 
-~> **NOTE:** This data source requires using the provider with Terraform Cloud and a Terraform Cloud 
-for Business account. 
-[Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing).
+~> **NOTE:** This resource requires using the provider with [Terraform Cloud](https://www.hashicorp.com/products/terraform/pricing).
```
Comment (brandonc) [inline website/docs/d/agent_pool.html.markdown]: Arguably, all the resources require Terraform Cloud... One bit of lost information here is that agent pools require a Cloud Business plan. Is your intent to improve the link text (by omitting the word "here", for instance)? I might suggest

```suggestion
~> **NOTE:** This resource requires using the provider with a [Terraform Cloud Business(https://www.hashicorp.com/products/terraform/pricing) account.
```

Comment (brandonc) [inline website/docs/d/agent_pool.html.markdown]: Arguably, all the resources require Terraform Cloud... One bit of lost information here is that agent pools require a Cloud Business plan. Is your intent to improve the link text (by omitting the word "here", for instance)? I might suggest

```suggestion
~> **NOTE:** This resource requires using the provider with a [Terraform Cloud Business(https://www.hashicorp.com/products/terraform/pricing) account.
```

**Review by brandonc — COMMENTED** [review]
Question

**Review by brandonc — APPROVED** [review]

**Comment by nfagerlund** [comment]
Ok, after talking in Slack we realized that agents have been working in TFE for YEARS, so closing this in favor of https://github.com/hashicorp/terraform-provider-tfe/pull/880

**Pull request closed without merging**